extensions/default/src/Toolbar/Toolbar.tsx

```diff
@@ -46,15 +47,18 @@ export default function Toolbar({ servicesManager }) {
         // These can... Trigger toolbar events based on updates?
         // Then sync using useEffect, or simply modify the state here?
         return (
-          <Component
-            key={id}
-            id={id}
-            {...componentProps}
-            bState={buttonState}
-            isActive={isActive}
-            onInteraction={args => ToolBarService.recordInteraction(args)}
-            servicesManager={servicesManager}
-          />
+          // The margin for separating the tools on the toolbar should go here and NOT in each individual component (button) item.
+          // This allows for the individual items to be included in other UI components where perhaps alternative margins are desired.
+          <div key={id} className={classnames('mr-1')}>
```

Comment:
This fixes the inconsistent spacing that was occurring is the previous rendition of the toolbar. Note that the spec says that there should be 4px between toolbar items.